torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -415,7 +414,6 @@ void RequestCallbackImpl::processPythonRRefFetchCall(
       Message m =
```

Comment:
I cannot comment on line 412, so add comments here:

```
result = std::make_shared<SerializedPyObj>(
            pythonRpcHandler.serialize(jit::toPyObject(rref->getValue())));
```

The above code will only block current streams instead of the ones in `lsctx`. I guess we need to add a `MultiStreamGuard` to wrap `getValue()`?